packages/components/src/card/card-media/index.js

```diff
@@ -8,7 +8,12 @@ import classnames from 'classnames';
  */
 import { MediaUI } from '../styles';
 
+/* eslint-disable jsdoc/valid-types */
+/**
+ * @param { import('../types').MediaProps & JSX.IntrinsicElements['div']} props
```

Comment:
I wonder if `MediaProps` is even necessary, considering that we're just using it to specify the `children` prop.

Reply:
Done in https://github.com/WordPress/gutenberg/pull/32561/commits/21a8c7c45ce3d3d671ccd19b2b89cdd16c2af0f1